design/crud-api/README.md

```diff
@@ -173,12 +147,13 @@ docker run --rm \
   -e NEO4J_USER=${NEO4J_USER} \
   -e NEO4J_PASSWORD=${NEO4J_PASSWORD} \
   -e MONGO_URI=${MONGO_URI} \
+  -e POSTGRES_TEST_DB_URI=${POSTGRES_TEST_DB_URI} \
```

Comment:
@Isuru-rangana why do we need `POSTGRES_TEST_DB_URI` can't we just use `POSTGRES_DB_URI` as the parameter name but use the value of the test DB URI.